.circleci/config.yml

```diff
@@ -86,11 +103,11 @@ workflows:
   version: 2
   Build and Test:
     jobs:
-      - Node.js 6:
```

Comment:
I couldn't decipher a reason why, but it seems that periods are no longer permitted in job names in CircleCI 2.1, prompting the removal of the period and a somewhat bizarre spelling of `NodeJS`.  The regex allows quite a few bits, but not periods!

```
[#/jobs/Node.js 10] string [Node.js 10] does not match pattern ^[A-Za-z][A-Za-z\\s\\d_-]*$
```